setup.py

```diff
@@ -29,7 +29,8 @@
 dev_requires = requires + [
     'bump2version',
     'mock',
-    'autopep8'
+    'autopep8',
+    'pytest-xdist ; sys_platform != "win32"'
```

Comment:
Is it really needed to use the `--forked` option (i.e. vs. just `-n2`)? If we could just use `-n2`, then the distributed test would also work on Windows. The state of the `--forked` option seems confused as well, as it has been spun out of xdist into pytest-forked, but that is listed as unmaintained.

Reply:
Originally, before I made changes to the code, and had only written tests, I used `--forked` to run the new tests to verify that the new tests passed when there was **one test per process** (because that would guarantee one kernel per test). `--forked` is the way I could see to do that easily. Now I have made changes to the code, `--forked` is not really necessary at all any more and could actually be removed.

After the subsequent optimization is applied, we will be more interested in just distributing the tests in some way like you say with half going somewhere and half going somewhere else. So would you accept your proposed change during that subsequent optimization work instead?

Reply:
https://github.com/jpmorganchase/nbcelltests/issues/107